app/controllers/user_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] Kindly rename this file to **`users_controller.rb`**

- [ ] Change the class name to **`UsersController`** for uniformity to avoid errors.

Reply:
Done, thanks